cmake/Platform/Common/RuntimeDependencies_common.cmake

```
@@ -83,6 +83,13 @@ function(ly_get_runtime_dependencies ly_RUNTIME_DEPENDENCIES ly_TARGET)
             if(is_imported AND is_system_library)
                 continue()
             endif()
+
+            # If the link dependency target is a shared library then
+            # add it as a runtime dependency as well.
+            get_target_property(link_dependency_type ${link_dependency} TYPE)
+            if(${link_dependency_type} STREQUAL "SHARED_LIBRARY")
```

Actually this should check the dependency against the list of all targets with runtime outputs, not just shared libraries.

```suggestion
            if(link_dependency_type IN_LIST LY_TARGET_TYPES_WITH_RUNTIME_OUTPUTS)
```

---

Thanks @lumberyard-employee-dm, I'll test your suggested change and update the PR if it works.

---

It worked! Changed. Thanks.